jobs/resource_job.go

```diff
@@ -268,16 +268,17 @@ type JobSettings struct {
 	GitSource *GitSource `json:"git_source,omitempty"`
 	// END Jobs + Repo integration preview
 
-	Schedule             *CronSchedule         `json:"schedule,omitempty"`
-	Continuous           *ContinuousConf       `json:"continuous,omitempty"`
-	Trigger              *Trigger              `json:"trigger,omitempty"`
-	MaxConcurrentRuns    int32                 `json:"max_concurrent_runs,omitempty"`
-	EmailNotifications   *EmailNotifications   `json:"email_notifications,omitempty" tf:"suppress_diff"`
-	WebhookNotifications *WebhookNotifications `json:"webhook_notifications,omitempty" tf:"suppress_diff"`
-	NotificationSettings *NotificationSettings `json:"notification_settings,omitempty"`
-	Tags                 map[string]string     `json:"tags,omitempty"`
-	Queue                *Queue                `json:"queue,omitempty"`
-	RunAs                *JobRunAs             `json:"run_as,omitempty"`
+	Schedule             *CronSchedule           `json:"schedule,omitempty"`
+	Continuous           *ContinuousConf         `json:"continuous,omitempty"`
+	Trigger              *Trigger                `json:"trigger,omitempty"`
+	MaxConcurrentRuns    int32                   `json:"max_concurrent_runs,omitempty"`
+	EmailNotifications   *EmailNotifications     `json:"email_notifications,omitempty" tf:"suppress_diff"`
+	WebhookNotifications *WebhookNotifications   `json:"webhook_notifications,omitempty" tf:"suppress_diff"`
+	NotificationSettings *NotificationSettings   `json:"notification_settings,omitempty"`
+	Tags                 map[string]string       `json:"tags,omitempty"`
+	Queue                *Queue                  `json:"queue,omitempty"`
+	RunAs                *JobRunAs               `json:"run_as,omitempty"`
+	Parameter            *JobParameterDefinition `json:"parameters,omitempty"`
```

Comment:
The field name needs to stay plural and its type needs to be an array:
```suggestion
	Parameters           []JobParameterDefinition `json:"parameters,omitempty"`
```